tests/includes/models/test-class-accredible-learndash-model-auto-issuance.php

```diff
@@ -30,7 +30,7 @@ public function test_get_results() {
 			'created_at'          => time(),
 		);
 		$data2 = array(
-			'kind'                => 'course_completed',
+			'kind'                => 'lesson_completed',
```

Comment:
The changes in this file do not increase any test coverage. In fact, all test cases pass without having`lesson_completed` in the `Accredible_Learndash_Model_Auto_Issuance::KINDS` array.

Instead, we should add another test assertion to `Accredible_Learndash_Model_Auto_Issuance::validate`,  for example, creating a new test case `test_validate_when_kind_is_lesson_completed` or adding lesson_completed data to `test_validate_when_creating` and `test_validate_when_updating`.

Reply:
Aah, ok. I go for the latter approach of adding `lesson_completed` to `...when_creating` and `...when_updating` tests.

Reply:
Decided to go with the first approach since it's easier to follow and maintain.